tests/system/providers/google/cloud/automl/example_automl_nl_text_classification.py

```diff
@@ -35,17 +35,14 @@
     AutoMLTrainModelOperator,
 )
 
-GCP_PROJECT_ID = os.environ.get("GCP_PROJECT_ID", "your-project-id")
-GCP_AUTOML_LOCATION = os.environ.get("GCP_AUTOML_LOCATION", "us-central1")
-GCP_AUTOML_TEXT_CLS_BUCKET = os.environ.get("GCP_AUTOML_TEXT_CLS_BUCKET", "gs://INVALID BUCKET NAME")
-
-# Example values
-DATASET_ID = ""
+ENV_ID = os.environ.get("SYSTEM_TESTS_ENV_ID")
+DAG_ID = "example_automl_classification"
+GCP_PROJECT_ID = os.environ.get("SYSTEM_TESTS_GCP_PROJECT", "default")
+GCP_AUTOML_LOCATION = "us-central1"
```

Comment:
Should we hardcode these values? Shouldn't it be fetched from env var?

Reply:
I was following the convention of the other system tests written, for example in https://github.com/apache/airflow/pull/30003 and also in the automl folder. Does it make sense to auto hard code them?

Reply:
cc: @VladaZakharova might help here.

Reply:
The main principle is to keep these tests self-sufficient. But they need an external resource and infra on which those resources run. We can control those resources, like creating a bucket on the fly, but if a user has a certain infra, IMO it would be best to give flexibility to the user to set up the test given the infra rather than looking at the test and setting up the infra. That would end up making these tests less useful wdyt?

Reply:
Nice point. Any suggestions that i can follow? It will help me tweak the DAG according to your use cases

Reply:
Generally, all the tests look good, except for the first point.

Reply:
Thanks. Do you mean to say that we should have it as: 
```
GCP_PROJECT_ID = os.environ.get("GCP_PROJECT_ID", "your-project-id")
GCP_AUTOML_LOCATION = os.environ.get("GCP_AUTOML_LOCATION", "us-central1")
GCP_AUTOML_TEXT_CLS_BUCKET = os.environ.get("GCP_AUTOML_TEXT_CLS_BUCKET", "gs://INVALID BUCKET NAME")
```
And so on?